src/components/dialogs/generator-modification-dialog.js

```diff
@@ -248,6 +253,106 @@ const GeneratorModificationDialog = ({
         clearable: true,
     });
 
+    const [isReactiveCapabilityCurveOn, isReactiveCapabilityCurveOnField] =
+        useBooleanValue({
+            label: 'ReactiveCapabilityCurve',
+            validation: { isFieldRequired: true },
```

Comment:
Maybe define somewhere `const fieldRequired = {isFieldRequired: true };` then use it here
This is to avoid hook rerender for nothing
same for 
```
{ disabled: true }
[ 'ActivePower',
  'MinimumReactivePower',
  'MaximumReactivePower',
]
```

Reply:
ok. done